lib/gooddata/lcm/lcm2.rb

```diff
@@ -420,7 +403,7 @@ def check_unused_params(actions, params)
           end
 
           table = Terminal::Table.new :headings => ['Action', 'Parameter', 'Description', 'Parameter Type'], :rows => rows
-          puts table.to_s
+          GoodData.logger.info("\n#{table.to_s}")
```

Comment:
warning: Lint/StringConversionInInterpolation: Redundant use of `Object#to_s` in interpolation. (rubocop)